ui/src/app/components/help-sidebar.tsx

```diff
@@ -156,6 +158,19 @@ export const HelpSidebar = withUserProfile()(
       }
     });
 
+    componentDidMount(): void {
+      participantStore.subscribe(participant => this.setState({participant}));
+    }
+
+    componentDidUpdate(prevProps: Readonly<Props>): void {
+      const {helpContent, hideSidebar} = this.props;
+      // close the sidebar on each navigation excluding navigating between participants in cohort review
+      if (hideSidebar > prevProps.hideSidebar &&
```

Comment:
If this is a boolean, we could probably do this as: `hideSidebar && !prevProps.hideSidebar`, if I am understanding this correctly?

Reply:
Yes, I've changed this after reworking state/props based on your comment below